wiki/Article_Style_Guide/en.md

```diff
@@ -1,192 +1,176 @@
+   - i.e. `.jpg` or `.osz`
 
-1.  As this wiki's purpose is to document facts, you should always avoid speculative and unsourced information. Generally speaking, information does not require sources if they can directly be seen in-game or are otherwise obvious. Other information however, such as information that is not widely known, must be sourced with a proper reference. Do *not* add content to an article if you cannot find a proper source.
-    1.  This does not applies to guides and tutorials which do not document facts but are here to help the userbase get a better understanding of the game mechanics and are, therefore, subjective. This does not mean that the content of guides and tutorials should not be verified.
+When noting a keyboard key, use capital letters for single characters and [Camel case](https://en.wikipedia.org/wiki/Camel_case) for modifers.
 
-2.  Articles in the main namespace should always be written in the third-person perspective and without terms referential to the reader. Try not to use abbreviations of words either. For instance, sentences like "If you miss an object when playing a beatmap, you will break your combo.." should be written as "Failing to hit an object while playing a beatmap will break the current combo.".
+When naming a file extension, add a period (`.`) then the file extension in lower cased letters.
 
-### Keeping articles concise and up to date
+## Links
 
-Articles should only contain information that is up to date, *i.e.*, implemented in the latest version of the game. Anything that is outdated should be moved to the History section of the article. When something changes, note the change in the History section and remove the outdated information from other sections of the article. It is unnecessary to mention when a particular feature was implemented; this is once again reserved for the History section of the article.
+Article links must follow these rules:
 
-### Grammar
+1. must use relative-absolute paths, unless linking to subfolder (then use relative-relative paths)
+  - i.e. `[Welcome](/wiki/Welcome/)`
+  - i.e. (from `Game_Modes` page) `[osu!mania](./osu!mania)`
+2. must be spelt correctly (links are case sensitive!!)
+3. must not specify the specific file name **regardless of language** (osu!wiki will handle this)
+  - do: `[Glossary](/wiki/Glossary/)`
+  - don't: `[Glossary](/wiki/Glossary/fr.md)`
+4. must not link to section headings (section headings on osu!wiki does not work)
 
-Pages on the wiki should only use *British English* as a matter of consistency. For instance, “color” should be “colour”, “center” should be “centre”, “skillful” should be “skilful”, “paralleled” should be “parallelled”, and “analyze” should be “analyse”. It is recommended to get accustomed to British English and American English spelling differences before writing an article.
+External links must follow these rules:
 
-### Capitalisation
+- prefer the `https` protocol, if available
```

Comment:
You do not really need to specify the https. Just

//osu.ppy.sh

will work, amazingly (See the Best_Of).